src/app/dataset-view/dataset-view-menu/dataset-view-menu.component.ts

```diff
@@ -167,7 +178,18 @@ export class DatasetViewMenuComponent implements OnInit, AfterViewInit {
     }
 
     private initClipboardHints(): void {
-        this.clipboardKamuCli = `kamu pull kamu.dev/${this.datasetBasics.alias}`;
-        this.clipboardKafka = `https://api.kamu.dev/kafka/${this.datasetBasics.alias}`;
+        this.clipboardReference = `https://kamu.dev/${this.datasetBasics.alias}`;
+        this.clipboardKamuCliPull = `kamu pull kamu.dev/${this.datasetBasics.alias}`;
+        this.clipboardKamuCliPush = `kamu push kamu.dev/${this.datasetBasics.alias}`;
+        this.clipboardKafka = `- coming soon -`;
+        this.clipboardRestUrlTail = `https://node.demo.kamu.dev/kamu/${this.datasetBasics.alias}/tail?limit=10`;
```

Comment:
We should un-hardcode `node.demo.kamu.dev` - backend host should come from api-server configuration we already use for GQL
(as a temporary solution until correct URLs are provided via API)

Reply:
Changed to  

> this.clipboardRestUrlTail = `${this.appConfigService.apiServerGqlUrl}/${this.datasetBasics.alias}/tail?limit=10`;